lifelines/fitters/aalen_additive_fitter.py

```diff
@@ -218,8 +217,9 @@ def _fit_static(self, dataframe, duration_col, event_col=None,
         self.durations = T
         self.event_observed = C
         self._compute_confidence_intervals()
+
         self.score_ = concordance_index(self.durations,
-                                        self.predict_median(dataframe).values.ravel(),
+                                        self.predict_cumulative_hazard(dataframe).values[-1,:].ravel(),
```

Comment:
not sure I want to keep this

Reply:
this changes the API, hence a version upgrade.

Reply:
Hmm I think it's useful but changing the version is a big change.